src/main/java/com/yahoo/sketches/theta/Sketch.java

```diff
@@ -27,13 +27,15 @@
 import com.yahoo.sketches.SketchesArgumentException;
 import com.yahoo.sketches.memory.Memory;
 
+import java.io.Serializable;
+
 /**
  * The top-level class for all sketches. This class is never constructed directly. 
  * Use the UpdateSketch.builder() methods to create UpdateSketches.
  * 
  * @author Lee Rhodes 
  */
-public abstract class Sketch {
+public abstract class Sketch implements Serializable {
```

Comment:
Thank you for your contribution!  
However, I have specifically avoided Java default serialization because it is so horribly inefficient. In a large system environment with literally billions of sketches this would reduce performance considerably.  Also,  Java default serialization will not work at all for the Direct* sketches, which are children of Sketch, since the data for these sketches is off the Java heap.  The preferred SerDe strategy is to use the toByteArray() methods and the Sketches methods heapify*() and wrap*(), which take a Memory object that wraps a byte array either on or off the Java heap.  Some big data systems, such as Hadoop/Pig provide mechanisms for efficiently transporting binary data.  For those that don't, It is much more efficient to encode the above binaries using Base64.  This can be done outside the context of the sketches themselves.  Because these transport capabilities vary across systems, the sketch library does the serialization and deserialization into these binary objects, the rest is up to the Systems Engineers to decide the most efficient way to do this.